src/api/user.js

```diff
@@ -46,8 +43,8 @@ const singInById = ({ id }) => (
     .catch(() => false)
 );
 
-const getItemList = () => (
-  axios.get('/itemList')
+const changeUserInfo = userData => (
```

Comment:
I'm not sure that you meant to replace `getItemList` with `changeUserInfo`. Is that so?